pages/docs/tutorials/run-rao.md

```diff
@@ -0,0 +1,436 @@
+range can be provided directly. For the initial tap and the tap-to-angle conversion map, it is easier to rely on
+an `IidmPstHelper` which fetches the information in the network.
+
+The usage rules must also be added to tell the RAO in which context the remedial can or must be applied.
+
+> For our example, let us create to usage rules:
+> - one _onInstant_ usage rule at the preventive instant making the PST range action available in the preventive instant
+> - one _onContingencyState_ usage rule for the already defined contingency to make the PST range action available after
+    said contingency occurred on the network
+
+```
+IidmPstHelper iidmPstHelper = new IidmPstHelper("BBE2AA1  BBE3AA1  1",network);
+
+crac.newPstRangeAction()
+    .withId("pst-range-action")
+    .withNetworkElement("BBE2AA1  BBE3AA1  1")
+    .withInitialTap(iidmPstHelper.getInitialTap())
+    .withTapToAngleConversionMap(iidmPstHelper.getTapToAngleConversionMap())
+    .newTapRange()
+        .withMinTap(-5)
+        .withMaxTap(10)
+        .withRangeType(RangeType.ABSOLUTE)
+        .add()
+    .newOnInstantUsageRule()
+        .withInstant("preventive")
+        .withUsageMethod(UsageMethod.AVAILABLE)
+        .add()
+    .newOnContingencyStateUsageRule()
+        .withInstant("curative")
+        .withContingency("contingency")
+        .withUsageMethod(UsageMethod.AVAILABLE)
+        .add()
+    .add();
+```
+
+### Create an auto topological action
+
+We can finish by adding a topological action to the CRAC, using the `newNetworkAction` method and
+the `newTopologicalAction` adder, and providing an action type (i.e. open or close) and a line on which apply this
+action. For illustrative purposes, let us make an auto remedial action from this remedial action. To do so, an
+_onContingencyState_ usage rule at the _auto_ instant must be defined with a _FORCED_ usage method.
+
+```
+crac.newNetworkAction()
+        .withId("topological-action")
+        .newTopologicalAction()
+            .withNetworkElement("FFR1AA1  FFR2AA1  1")
+            .withActionType(ActionType.OPEN)
+            .add()
+        .newOnContingencyStateUsageRule()
+            .withInstant("auto")
+            .withContingency("contingency")
+            .withUsageMethod(UsageMethod.FORCED)
+            .add()
+        .add();
+```
+
+We are almost there! The CRAC is complete and the RAO is almost ready to run. Let us see how to do this in the following
+section.
+
+## Run the RAO
+
+A last pre-processing step is required to run the RAO. The network and the CRAC must be processed together as
+a `RaoInput`. This is achieved through a `RaoInputBuilder`.
+
+```
+RaoInput.RaoInputBuilder raoInputBuilder = RaoInput.build(network,crac);
+```
+
+To run, the RAO also requires `RaoParameters` to properly set its desired behaviour. For our example, the default
+parameters will be fully sufficient.
```

Comment:
add a sentence with a link to the parameters page "See how you can tune the RAO using the parameters here"